README.md

```diff
@@ -1,11 +1,11 @@
 <div style="text-align: center">
-<img src="https://huggingface.co/datasets/trl-internal-testing/example-images/resolve/main/images/trl_banner_dark.png">
+<img src="https://huggingface.co/datasets/trl-internal-testing/example-images/resolve/main/images/trl_banner_dark.png" alt="TRL Banner">
 </div>
 
 # TRL - Transformer Reinforcement Learning
 
 <h3 align="center">
-    <p>Full stack library to post-train large language models.</p>
+    <p>A comprehensive library to post-train large language models (LLMs).</p>
```

Comment:
nit: since we also support training VLMs, it's probably time to use this instead :)

```suggestion
    <p>A comprehensive library to post-train foundation models</p>
```